### PR TITLE
Update readme to better reflect open docs

### PR DESCRIPTION
**Pull request by iamnbutler**
Now that the docs are open sourced we should better reflect that people can contribute changes and help us evolve the docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
+The docs are written in Markdown. Each file corresponds to a page in the documentation. The documentation is hosted on [GitBook](https://www.gitbook.com/) and can be accessed at [docs.zed.dev](https://docs.zed.dev/general/readme). The SUMMARY.md file in this repository defines the structure of the documentation.
 
-### Configure Zed
 
-Use `CMD + ,` to open your custom settings to set things like fonts, formatting settings, per-language settings and more. You can access the default configuration using the Zed > Preferences > Open Default Settings menu item. See Configuring Zed for all available settings.
+### Previewing Changes
+Currently, there is no way to build the documentation locally and preview your changes locally. However, if you stick to simple Markdown, most things should render correctly. 
 
-### Set up your key bindings
+You can Github’s preview or a tool like [markdown live preview](https://markdownlivepreview.com/) to preview your changes.
```
Comment (petros) [inline README.md:23]:
```suggestion
You can use GitHub's preview or a tool like [markdown live preview](https://markdownlivepreview.com/) to preview your changes.
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,40 @@
-# Getting Started
+# Zed Docs
+Welcome to the Zed Docs repository! 
 
-Welcome to Zed! We are excited to have you. Here is a jumping-off point to getting started.
+We highly encourage contributions from the community to improve and expand the documentation. Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter.. Our goal is to build the world's best text editor for developers and their teams.
```
Comment (petros) [inline README.md:4]:
```suggestion
We highly encourage contributions from the community to improve and expand the documentation. Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter. Our goal is to build the world's best text editor for developers and their teams.
```

**Comment by iamnbutler** [comment]
@petros I just realized that this will replace the Getting Started page on gitbook. Is there a way for the github README to be seperate from the doc you land on on docs.zed.dev?

File: README.md [hunk 3]
```diff
@@ -1,19 +1,52 @@
...
+We highly encourage contributions from the community to improve and expand the documentation.
 
-### Download Zed
+## Contributing
 
-If you have been invited to try Zed, head to Releases and download Zed. Once you do this the first time, the app will auto-update to keep you on the most recent version.
+Your contributions are invaluable to us, and we appreciate your interest in helping us improve the Zed documentation. Whether you want to fix typos, improve explanations, or add new content, every contribution is welcome.
 
-### Configure Zed
+For more on contributing see [CONTRIBUTING.md](https://github.com/zed-industries/docs/blob/main/CONTRIBUTING.md)
```
Comment (PineappleRind) [inline README.md:13]: minor nitpick: comma after "contributing"

File: README.md [hunk 4]
```diff
@@ -1,19 +1,52 @@
...
+
+### Updating the Table of Contents
+
+When adding a new file or proposing changes to the structure of the documentation, please edit the SUMMARY.md file, which serves as the table of contents for the documentation.
+
+## Join the Zed Community
+
+We're on a mission to create a better way to write software, and this is just the beginning.
+
+By contributing to the Zed Docs you're joining a community of passionate developers and helping to shape the future of software development.
```
Comment (PineappleRind) [inline README.md:48]: minor nitpick: comma after Docs

**Comment by JosephTLyons** [comment]
I think the few nits @PineappleRind mention are good.  That being said, I don't know if we can change this without affecting the docs landing page.  Will wait to hear from Petros.

**Review by JosephTLyons — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,19 +1,52 @@
...
+
+You can join our [discord](https://discord.gg/SSD9eJrn6s) to discuss docs, zed, any anything else text editing.
```
Comment (PineappleRind) [inline README.md:50]: one more thing I didn't catch earlier —
```suggestion
You can join our [Discord](https://discord.gg/SSD9eJrn6s) to discuss docs, Zed, and all things text editing.
```

**Comment by iamnbutler** [comment]
Yeah I think we'll need to unify this and the Getting Started page from my earlier conversation with Petros. I'll leave this open until I have time to do it or someone wants to take a stab at it.

**Pull request closed without merging**